docs/src/components/GuidelinesBuilder/GuidelinesPageBuilder.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+@import '~bpk-mixins';
+
+.bpk-docs-guidelines-page {
+  &__image {
+    height: $bpk-spacing-lg * 10;
+    color: $bpk-color-sky-gray-tint-01;
+  }
+
+  &__section {
+    padding: $bpk-spacing-xl * 2 $bpk-spacing-md * 10 $bpk-spacing-xs * 13 $bpk-spacing-xs * 10;
+    background-color: $bpk-color-white;
+
+    &--alternate {
+      background-color: $bpk-color-sky-gray-tint-06;
+    }
+
+    &--image {
+      height: $bpk-spacing-sm * 41;
+      color: $bpk-color-sky-gray-tint-01;
+    }
+  }
+
+  &__heading {
+    margin-bottom: 0;
+    color: $bpk-color-sky-gray;
+    text-align: left;
+  }
+
+  &__markdown {
+    width: auto;
+    color: $bpk-color-sky-gray-tint-02;
+
+    @include bpk-breakpoint-mobile {
+      width: $bpk-spacing-xl * 20;
+    }
+  }
+
+  &__navigation {
+    display: flex;
+    padding: $bpk-spacing-xs * 10;
+    padding-bottom: $bpk-spacing-xs;
+    justify-content: space-between;
+    background-color: $bpk-color-monteverde;
+    color: $bpk-color-white;
+    text-decoration: none;
+
+    &--icon {
+      padding-top: $bpk-spacing-xs * 4.7;
```

Comment:
This seems like a very random value. We should stick to combinations of whole tokens...

Reply:
Yeah it didn't quite meet the design specs without it being this so 🤷‍♂

Reply:
In that case the design specs are wrong!!

Reply:
Turns out I was wrong 🤦‍♂